README.md

```diff
@@ -20,27 +20,46 @@ There is a `STEERING_LKA`-ish message and more in some new Toyotas that currentl
 
 ## Cars
 
-### Successfully running openpilot
+### 🟢 Successfully running openpilot
 
 These cars can run openpilot but are not listed on https://comma.ai/vehicles or [CARS.md](https://github.com/commaai/openpilot/blob/master/docs/CARS.md) because Comma (the company) understandably doesn't want to own the security key hacking process. Follow the [Setup Guide](#setup-guide) below and you'll have it working.
 
 * 2021-2023 RAV4 Prime
+  * Upstreamed into openpilot's master branch.
 * 2021-2023 Sienna
-* 2020+ Yaris Hybrid (EUDM/JDM/MXDM)
-  * Needs extra steps. See [Support Status Overview](#support-status-overview).
-
-### May be possible to hack but hasn't been tried
+  * Upstreamed into openpilot's master branch.
+* 2020-2022 Yaris Hybrid (EUDM/JDM/MXDM)
+  * Memory dump hack works but the key is not in the same address as RAV4 Prime.
+  * Brute force efforts to find key location successful on both Euro and Japanese. Euro gave up due to broken C3.
```

Comment:
`Brute force efforts to find key location successful on both European and Japanese Yaris Hybrid. European user eventually gave up due to unrelated broken C3 device issue.`